Mathlib/Topology/Algebra/Module/PerfectPairing.lean

```diff
@@ -0,0 +1,91 @@
+/-
+Copyright (c) 2025 Yaël Dillies, Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Andrew Yang
+-/
+import Mathlib.LinearAlgebra.BilinearMap
+import Mathlib.Topology.Algebra.Module.LinearMap
+
+/-!
+# Continuous perfect pairings
+
+This file defines continuous perfect pairings.
+
+For a topological ring `R` and two topological modules `M` and `N`, a continuous perfect pairing is
+a continuous bilinear map `M × N → R` that is bijective in both arguments.
+
+Note that this does **not** imply that the dual of `M` is `N` and vice-versa (one would instead need
```

Comment:
Should you say topological dual? Or is this clear from the context.

Reply:
Good point, clarified